opencensus/trace/span.py

```diff
@@ -176,6 +176,16 @@ def add_attribute(self, attribute_key, attribute_value):
         """
         self.attributes[attribute_key] = attribute_value
 
+    def add_annotation(self, description, **attrs):
+        """Add an annotation to span.
+
+        :type description: str
```

Comment:
nit: please update the documentation like below,
```
:type description: str
:param description: A user-supplied message describing the event.
                    The maximum length for the description is 256 bytes.

:type attrs: dict
:param attrs: keyworded arguments e.g. failed=True, name='Caching'
```

Reply:
Nice doc update request, let me do that, thank you!

Reply:
Although the `type attrs: dict` perhaps isn't that accurate as they are keyworded arguments. Passing in a dict d (without unfurling it i.e. **d) will register that as a non-keyworded argument :)

Reply:
Right, I was thinking what type is accurate to describe the keyword arguments, but cannot think of a better one. `kwargs` is fine.

Reply:
Cool.